Neos.Flow/Classes/Annotations/Route.php

```diff
@@ -0,0 +1,76 @@
+<?php
+namespace Neos\Flow\Annotations;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+/**
+ * Used to configure Routes from a method
+ *
+ * @Annotation
+ * @Target("METHOD")
+ */
+final class Route
+{
+    /**
+     * Name
+     *
+     * @var string|null
+     */
+    public $name;
+
+    /**
+     * HTTP Methods
+     *
+     * @var array|null
+     */
+    public $httpMethods;
+
+    /**
+     * URI Pattern
+     *
+     * Example: GRANT
+     *
+     * @var string
+     */
+    public $uriPattern;
+
+    /**
+     * Format
+     *
+     * Example: html
+     *
+     * @var string|null
+     */
+    public $format;
+
+    /**
+     * Append Exceeding Arguments
+     *
+     * @var bool|null
+     */
+    public $appendExceedingArguments;
+
+    /**
+     * @param array $values
+     */
+    public function __construct(array $values)
+    {
+        if (!isset($values['uriPattern'])) {
```

Comment:
Let's make `uriPattern` available as anonymous argument too?
```suggestion
        if (!isset($values['value']) && !isset($values['uriPattern'])) {
```

Reply:
Do we have that in a existing annotation i can copy/paste from?

Reply:
Sure, just take a look at e.g. `IgnoreValidation` - or any other Annotation that takes arguments. Most of them have one argument defined as "anonymous". You can also just commit my suggestions